src/encoded/static/components/collection.js

```diff
@@ -109,22 +141,58 @@ class FacetChart extends React.Component {
 
     render() {
         const { chartId, facet } = this.props;
+        let terms = [];
+        let otherTerms;
+
+        // If we have more facet terms than we allow to be displayed (determined through
+        // MAX_CHART_ITEMS), then just get MAX_CHART_ITEMS to display as usual, and collect the
+        // rest to display as "other."
+        if (facet.terms.length > MAX_CHART_ITEMS) {
+            // We have more facet terms than we display individually. First get the terms to
+            // display as individual items.
+            terms = facet.terms.slice(0, MAX_CHART_ITEMS);
+
+            // Now collect the terms to display as "other."
+            otherTerms = facet.terms.slice(MAX_CHART_ITEMS);
```

Comment:
If there are 10 items we'll end up with an "other" wedge that only represents a single item, which seems non-ideal. So maybe slice at MAX_CHART_ITEMS - 1 (which basically reinterprets MAX_CHART_ITEMS as the max *including* the "other" item).